src/main_app/jobs_workers/admin_jobs_workers/crop_main_files/upload.py

```
@@ -18,7 +18,7 @@
 def upload_cropped_file(
     cropped_filename: str,
     cropped_path: Path,
-    site: mwclient.Site | None,
+    site: Site | None,
     wikitext: str = None,
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)